src-electron/generator/matter/chip-tool/templates/helper.js

```diff
@@ -163,13 +163,27 @@ async function assertSameTestType(current, expected) {
   throw error;
 }
 
+const dep = templateUtil.deprecatedHelper;
+
 //
 // Module exports
 //
-exports.asDelimitedCommand = asDelimitedCommand;
-exports.asTypeMinValue = asTypeMinValue;
-exports.asTypeMaxValue = asTypeMaxValue;
-exports.structs_with_cluster_name = structs_with_cluster_name;
+exports.asDelimitedCommand = dep(
+  asDelimitedCommand,
+  'asDelimitedCommand has been deprecated. Use cleanse_label_as_kebab_case instead'
+);
+exports.asTypeMinValue = dep(
```

Comment:
We have gotten into a habit of deprecating already created helpers and not removing them because that maintains backwards compatibility. For eg: A customer has an old release of matter(with old helpers) and updates zap then things continue to work for them instead of breaking. However they will get these warnings which will let them know that they need to update their templates/helpers.

However I am starting to see a large accumulation of deprecated helpers so we may need a plan on how to deprecate old helpers.  May be remove all deprecated helpers after 2-3 releases of the stack but this can annoy customers. +@tecimovic